schema.sql

```diff
@@ -6,3 +6,14 @@ CREATE TABLE animals (
   neutered BOOLEAN,
   weight_kg DECIMAL(5,2)
 );
+
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(30),
+  age INTEGER
+);
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(30)
+);
```

Comment:
- [ ] It's obvious you did alter the `Animals` table, but as required you need to document the changes made here.

e.g.

```
   ALTER TABLE animals DROP column species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```